utils/pubEdge/relations.js

```diff
@@ -76,3 +76,30 @@ const createRelationTypeEnum = () => {
 
 export const relationTypes = Object.keys(relationTypeDefinitions);
 export const RelationType = createRelationTypeEnum();
+
+const findParentEdge = (pubEdges, relationTypes, inbound) => {
+	for (let i = 0; i < pubEdges.length; i++) {
+		const pubEdge = pubEdges[i];
+		const { pubIsParent, relationType } = pubEdge;
+
+		if (inbound ? pubIsParent : !pubIsParent) {
+			for (let j = 0; j < relationTypes.length; j++) {
+				if (relationType === relationTypes[j]) {
+					return pubEdge;
+				}
+			}
+		}
+	}
+
+	return null;
+};
+
+export const findParentEdgeByRelationType = (pub, ...relationTypes) => {
```

Comment:
I tend to over-use rest parameters because I like the reduction in visual noise by removing the extra pair of square brackets. But you're right, it's not really adding much here.